src/core/thread/osthread.d

```diff
@@ -820,6 +820,7 @@ class Thread
                     if ( pthread_create( &m_addr, &attr, &thread_entryPoint, cast(void*) this ) != 0 )
                         onThreadError( "Error creating thread" );
                 }
+                assert( pthread_attr_destroy( &attr ) == 0 );
```

Comment:
This will get removed when asserts are disabled. Please write it as `pthread_attr_destroy( &attr ) && assert(false);` or code to that effect.

Reply:
Ah, missed! Thanks.